28-04/task_2/task_2_second_version.rb

```diff
@@ -1,22 +1,16 @@
-File.open("palindrom.txt").each do |line|
-  pal = line.strip.downcase.delete("/\s\.\,/")
-  if pal == pal.reverse
-    puts "True"
-  else
-    puts "False"
-  end
-end
-
-
-
 def palindrome?(string)
   if string.length == 1 || string.length == 0
-    true
+    puts "True"
   else
     if string[0] == string[-1]
       palindrome?(string[1..-2])
     else
-      false
+      puts "False"
     end
   end
 end
+
+File.open("palindrom.txt").each do |line|
+  str_punct_marks_off_downcase = line.strip.downcase.delete("/\s\.\,/")
```

Comment:
since `#delete` removes all occurrences of symbol in string, you can omit `/` leaving only `"\s\.\,"` in a pattern